odk1-src/form-functions.rst

```diff
@@ -0,0 +1,502 @@
+
+  Returns the natural log of ``number``.
+
+.. function:: log10(number)
+
+  Returns the base-10 log of ``number``.
+
+.. function:: abs(number)
+
+  Returns the absolute value of ``number``.
+
+.. function:: sin(number)
+
+  Returns the sine of ``number``.
+
+.. function:: cos(number)
+
+  Returns the cosine of ``number``.
+  
+.. function:: tan(number)
+
+  Returns the tanget of ``number``.
+
+.. function:: asin(number)
+
+  Returns the arc sine of ``number``.
+  
+.. function:: acos(number)
+
+  Returns the arc cosine of ``number``.
+
+.. function:: atan(number)
+
+  Returns the arctan of ``number``.
+
+.. function:: atan2(y,x)
+
+  Returns the multi-valued inverse tangent of ``y``, ``x``.
+
+.. function:: sqrt(number) 
+
+  Returns the square root of ``number``.
+
+.. function:: exp(x) 
+
+  Returns ``e^x``.
+
+.. function:: exp10(number arg)
+
+  Returns ``10^x``.
+
+.. function:: pi()
+
+  Returns an approximation of the mathematical constant π.
+
+
+Control flow
+--------------
+
+.. function:: if(expression, then, else)
+
+  Returns ``then`` if ``expression`` evaluates to ``True``. 
+  Otherwise, returns ``else``.
+
+  
+.. function:: not(arg)
+
+  Returns the opposite of ``boolean(arg)``.
+
+  
+.. function:: coalesce(arg, arg) 	
+
+  Returns first non-empty value of the two arguments.
+  Returns an empty string if both arguments are empty or non-existant.
+
+  
+.. function:: true()
+
+  Evaluates to ``True``.
+
+.. function:: false()
+
+  Evaluates to ``False``.
+  
+.. function:: boolean(arg) 
+
+  Returns ``True`` if ``arg`` is:
+  
+  - a number other than zero
+  - a non-empty string
+  - a non-empty collection
+  - a comparison or expressions that evaluates to ``True``.
+   
+  Returns ``False`` if ``arg`` is:
+  
+  - the number 0
+  - an empty string
+  - an empty collection
+  - a comparison or expression that evaluates to ``False``.
+
+  
+  
+Date and time
+----------------
+
+
+.. function:: today()
+
+  Returns the current date without a time component.
+
+.. function:: now()
+
+  Returns the current datetime in the current time zone.
+
+  
+Converting dates and time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  
+ 
+.. function:: decimal-date-time(dateTime)
+
+  Converts ``dateTime`` value to the number of days since January 1, 1970 UTC. This is the format used by Excel.
+
+.. function:: decimal-time(time)
+
+  Converts ``time`` to a number representing a fractional day.
+  For example, noon is 0.5 and 6pm is 0.75.
+
+.. function:: date(days)
+
+  Converts an integer representing a number of ``days`` from 01 January 1970 (the `Unix Epoch`_) to a standard date value.
+  
+.. _Unix Epoch: https://en.wikipedia.org/wiki/Unix_time
+
+
+Formatting dates and times for display
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
+  
+  
+.. function:: format-date(date, format)
+
+  Returns ``date`` as a string formatted as defined by ``format``.
+  
+  The following identifiers are used in the ``format`` string:
+  
+  ``%Y``
```

Comment:
Can we put this stuff in a table? Current layout feels wasteful.

Reply:
i like the semantics of definition lists, so maybe a compact view...